.github/workflows/manual-detached-test.yml

```diff
@@ -10,6 +10,7 @@ jobs:
         with:
           limit-access-to-actor: true
           detached: true
+          connect-timeout: 60
```

Comment:
Could we call this `connect-timeout-seconds`? Otherwise it is unclear whether this is in seconds, minutes or even milliseconds.

Reply:
I was honestly thinking the exact same thing this morning. Thanks for the suggestion!